src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/Data/apps/tdd/endring-av-navn/options/carbrands.json

```diff
@@ -1,11 +1,19 @@
- [
-    {
-      "value": "01",
-      "label": "Audi"
-    },
-    {
-      "value": "02",
-      "label": "BMW"
-    }
- ]
+[
```

Comment:
maybe the test should include a fifth element where it is a duplicate of the first to assert that you actually are dropping the new key-value-pair?

Reply:
It shouldn't matter which one we drop if they are the same.

The first element is now identical to the forth, what would a fifth element do for us?

Reply:
Seems to me that the test only asserts if it can add n extra elements, but your change does a check if there is duplicates.
There'are no test that checks if the duplicate handling happens that I can see.

Reply:
You're right. It's difficult to see from the test if the change is being tested.

I ran the tests before and after my change so I feel safe to say that the fix prevents the error it was targeting. I have also actually tested this in the nbib/digital-pliktavlevering app.